### PR TITLE
Vampyr needs getNodeTableSize and getNodesAtDepth

### PR DESCRIPTION
**Pull request by bjorgve**

### CODE REVIEW

**Review by robertodr — APPROVED** [review]

File: src/trees/MWTree.h [hunk 1]
```diff
@@ -103,6 +103,9 @@ template <int D> class MWTree {
     void makeNodeTable(MWNodeVector<D> &nodeTable);
     void makeNodeTable(std::vector<MWNodeVector<D>> &nodeTable);
 
+    int getNodeTableSize() const { return this->endNodeTable.size(); }
```
Comment (stigrj) [inline src/trees/MWTree.h:106]: I think we already have `getNEndNodes()` ?

Comment (bjorgve) [reply src/trees/MWTree.h:106]: Oh god yes, didn't see that!

File: src/trees/MWTree.h [hunk 2]
```diff
@@ -103,6 +103,9 @@ template <int D> class MWTree {
...
+    int getNNodesAtDepth(int i) const { return this->nodesAtDepth[i]; }
```
Comment (stigrj) [inline src/trees/MWTree.h:107]: and `getNNodes(int depth)`...

Comment (bjorgve) [reply src/trees/MWTree.h:107]: oh god yes again! I'll kill this one and update the vampyr counterpart

**Pull request closed without merging**